docs/bitcoin-core-usage.md

```diff
@@ -14,10 +14,6 @@ We are not liable for any coins that may be lost through this method. The softwa
 
 This method of using hardware wallets uses Bitcoin Core as the wallet for monitoring the blockchain. It allows a user to use their own full node instead of relying on an SPV wallet or vendor provided software.
 
-However, Bitcoin Core needs a few changes for this work.
```

Comment:
We should still note which versions/commit support this. Once 0.18 is released we should recommend that release or anything beyond.

Reply:
Done